data-and-functions-1/src/functions/getUserById.js

```diff
@@ -0,0 +1,19 @@
+const getUserById = (data, userId) => {
+  if (userId == null) {
+    return null
+  }
+  if (data == null) {
+    return null
+  }
+  if (data.users == null) {
+    return null
+  }
```

Comment:
See if you can combine these three tests into one `if` using the boolean OR operator `||`